scripts/sct_deepseg_lesion.py

```diff
@@ -17,104 +17,125 @@
-                      example=["0", "1"],
-                      default_value="1")
-    parser.add_option(name="-ofolder",
-                      type_value="folder_creation",
-                      description="output folder.",
-                      mandatory=False,
-                      example="My_Output_Folder/",
-                      default_value="")
-    parser.add_option(name="-r",
-                      type_value="multiple_choice",
-                      description="remove temporary files.",
-                      mandatory=False,
-                      example=['0', '1'],
-                      default_value='1')
-    parser.add_option(name="-v",
-                      type_value="multiple_choice",
-                      description="1: display on (default), 0: display off, 2: extended",
-                      mandatory=False,
-                      example=["0", "1", "2"],
-                      default_value="1")
-    parser.add_option(name='-igt',
-                      type_value='image_nifti',
-                      description='File name of ground-truth segmentation.',
-                      mandatory=False)
+
+    parser = argparse.ArgumentParser(
+        description='MS lesion Segmentation using convolutional networks.'
+                    '\n\nReference: C Gros, B De Leener, et al. '
+                    'Automatic segmentation of the spinal cord and intramedullary multiple sclerosis lesions with '
+                    'convolutional neural networks (2018). arxiv.org/abs/1805.06349',
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py"))
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        "-i",
+        help='input image. (e.g.,"t1.nii.gz")',
+        metavar = Metavar.file)
+    mandatory.add_argument(
+        "-c",
+        help='type of image contrast. \nt2: T2w scan with isotropic or anisotropic resolution.'
+             ' \nt2_ax: T2w scan with axial orientation and thick slices.'
+             ' \nt2s: T2*w scan with axial orientation and thick slices.',
+        choices=('t2', 't2_ax', 't2s'))
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="show this help message and exit")
+    optional.add_argument(
+        "-centerline",
+        help="Method used for extracting the centerline."
+             "\nsvm: automatic centerline detection, based on Support Vector Machine algorithm."
+             "\ncnn: automatic centerline detection, based on Convolutional Neural Network."
+             "\nviewer: semi-automatic centerline generation, based on manual selection of a few "
+             "points using an interactive viewer, then approximation with NURBS."
+             "\nfile: use an existing centerline by specifying its filename with flag -file_centerline (e.g. -file_centerline t2_centerline_manual.nii.gz).\n",
```

Comment:
i would remove the example "-file_centerline t2_centerline_manual.nii.gz" (it does not go here)

Reply:
Ok sounds good! Changed in commit [bb94b78](https://github.com/neuropoly/spinalcordtoolbox/commit/bb94b789633a29b858d487bd5a73fdac26ec738f)